pypit/scripts/run_pypit.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+#
+# See top-level LICENSE file for Copyright information
+#
+# -*- coding: utf-8 -*-
+
+
+"""
+This script pushes a FITS file to ginga
+"""
+from __future__ import (print_function, absolute_import, division,
+                        unicode_literals)
+
+try:
+    from xastropy.xutils import xdebug as debugger
+except:
+    import pdb as debugger
+
+def parser(options=None):
```

Comment:
I'm open to using argparse, but it seems that we've lost the -h option (which displays the msgs.usage). It would be good if the usage could be displayed (as it lists the available spectrographs and reduction algorithms).